sci-biology/MuSeqBox/MuSeqBox-5.7.2021.ebuild

```diff
@@ -1,20 +1,19 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit toolchain-funcs
 
 DESCRIPTION="Parse and filter BLAST output into tab-delimited file"
 HOMEPAGE="http://brendelgroup.org/bioinformatics2go/MuSeqBox.php"
 SRC_URI="http://www.brendelgroup.org/bioinformatics2go/Download/MuSeqBox-${PV//./-}.tar.gz"
-# code will move to https://github.com/BrendelGroup
 
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64"
 
-S="${WORKDIR}/MUSEQBOX5.5"
+S="${WORKDIR}/MUSEQBOX5.7"
```

Comment:
Here we can use on the version manipulation functions to make future version bumps a bit easier, e.g.:
```
S="${WORKDIR}/MUSEQBOX$(ver_cut 1-2)"
```
See here for the full overview of all available version manipulation functions: https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#version-manipulation-and-comparison-commands